### ISSUE TITLE
Krane 1.0 release

### ISSUE THREAD

**Issue by dturn** [Shopify/krane#528]
## Problem
We're transitioning from kubernetes-deploy to krane. We need to update the gemspec, version to 1.0, and deploy to ruby gems under `krane`. This PR should be completed immediately after #527.

## Possible solution
Update the gemspec, shipit.yml, and Version file. https://github.com/Shopify/kubernetes-deploy/pull/631

## Acceptance criteria

- [x] Audit our public interfaces to ensure that arguments to each public function are sensible and consistent (same names for same thing, required vs. optional, on initialize vs on run). (https://github.com/Shopify/kubernetes-deploy/pull/619)
- [x] Delete everything "kubernetes-deploy" (https://github.com/Shopify/kubernetes-deploy/pull/622)
- [x] StatsD Rename (https://github.com/Shopify/kubernetes-deploy/pull/623)
- [x] Ensure we have documented what our public interfaces are (and possibly what they are not) (https://github.com/Shopify/kubernetes-deploy/pull/634)
- [x] Ensure that all 1.0 tasks are complete or intentionally deferred.
- [ ] Datadog dashboards are updated for `krane` and `kubernetes-deploy`
- [x] Migration guide is linked near the top of the readme. (https://github.com/Shopify/kubernetes-deploy/pull/634)
- [x] `gem 'krane', '~> 1.0.0'` works. 

## Additional context
https://github.com/Shopify/kubernetes-deploy/issues/256#issuecomment-475580770

**Comment by dturn** [comment]
As we transition to Krane 1.0. We'll create 1 or more kubernetes-deploy v1.0.0-pre.X releases. The final pre-release version will be a transition version that maintains backwards compatibility with kubernetes-deploy as well as have all of the Krane commands. 

Krane 1.0.0 will not include any kubernetes-deploy modules or commands. We'll be publishing a transition guide.

**Comment by dturn** [comment]
Krane 1.0.0 has been released.
